global-state/jar/build.boot

```diff
@@ -0,0 +1,91 @@
+(def +version+ "3.37-SNAPSHOT")
+
+(set-env!
+  :project 'com.sixsq.slipstream/globalstate
```

Comment:
I would propose using "metering" to replace "global state" everywhere.  This probably better describes what the service does.  This change should be carried throughout all of the files. (Apologies for the late opinion on the naming.)

Reply:
Done